content/dips/DIP-18.md

```diff
@@ -0,0 +1,88 @@
+
+To keep up with the competitive landscape, the DAO will also need to operate quickly. Doing so will require a standardized process for launching new subDAOs in charge of important functions. The playbook to be written by the Ops Trust will give the community a thoughtful approach for quickly spinning up new subDAOs, including all legal and operational procedures necessary.
+
+## Specification
+
+**The subDAO will be funded for a period of 6 months**
+
+Reverie will work with outside legal counsel to establish a Guernsey Purpose Trust. The Trust will be launched with the following contributors:
+
+**Trustees**
+- Reverie Reserves, LLC
+- Joanna Pope
+- Callen Van Den Elst
+
+**Enforcer**
+George Beall
+
+**Responsibilities**
+- Open a fiat bank account for the Ops Trust
+- Hire and manage external legal staff for ongoing guidance
+- Payments and Financial Reporting
+- Create communication channels for the Ops Trust, as needed, which can serve as the standard dYdX DAO’s communication channel going forward
+- Build and share a DAO playbook for launching new dYdX subDAOs
+
+**Compensation and Expenses**
+- Reverie will receive **$15,000 per month** and a **$90,000 success fee** for publishing the DAO playbook
+- Joanna Pope will receive **$1,500 per month** as Trustee
+- Callen Van Den Else has waived compensation as Trustee
+- George Beall will receive **$2,000 per month** as Enforcer
+
+We are requesting an additional **$100,000** to cover legal and operational expenses for all Ops Trust related engagements.
+
+In total, $360,000 will be used to fund the initial 6 month operation of the Ops subDAO.
+
+**DYDX Amount**
+
+The amount of DYDX to be moved from the community treasury to the Ops Trust Multisig will be determined using a 24h vwap of the trades executed on the most liquid exchange, namely Binance.com. The previous day's trades will be used to capture a full 24h window so as to avoid timing constraints. The following data file is used: https://data.binance.vision/data/spot/daily/trades/DYDXUSDT/DYDXUSDT-trades-2022-07-17.zip
+
+This formula will be used to calculate the price: Σ(Price * Volume) / Σ(Volume).
+
+From the data, we find: $X / X = $X.
+
+The DYDX Amount to be transferred will be $360,000 / $X = **X** (rounded up to the nearest thousand for simplicity).
+
+
+## Rationale
+
+The community has addressed certain concerns and issues with a dYdX Operations subDAO run by Reverie in the DRC, but overall consensus has been in strong support of launching. The dYdX Ops subDAO will act as launching pad for the growth of a subDAO run dYdX DAO.
```

Comment:
"The community has addressed certain concerns and issues with a dYdX Operations subDAO run by Reverie in the DRC, but overall consensus has been in strong support of launching."

A little unclear could you please revise.